client/app/components/onboardingGuide/GuideStatusPage.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideStatusPage = (props) => {
+
```

Comment:
It's possible - static might not be available, but that's just a shorthand to some special property magic. Anyway, I added those PropTypes.